service/tpo/tpo.go

```diff
@@ -98,15 +98,23 @@ func (s *Service) Ensure(TPO *draughtsmantpr.CustomObject) error {
 
 	_, err := s.Get()
 	if IsNotFound(err) {
+		s.logger.Log("debug", "creating TPO")
+
 		endpoint := s.draughtsmanTPR.Endpoint(DefaultNamespace)
 		b, err := s.k8sClient.Core().RESTClient().Post().Body(TPO).AbsPath(endpoint).DoRaw()
 		if err != nil {
 			return microerror.Maskf(err, string(b))
 		}
+
+		return nil
 	} else if err != nil {
 		return microerror.Mask(err)
 	}
 
+	s.logger.Log("debug", "updating TPO")
+
+	TPO.ObjectMeta.ResourceVersion = ""
```

Comment:
The patch was broken because of the resource version we got when fetching the custom object beforehand. When updating it we have to reset it.